.gitignore

```diff
@@ -0,0 +1,4 @@
+**/__pycache__/*
+environment
```

Comment:
@laemtl the environment file should not be in gitignore in case changes need to be made to that file. Instead, I would recommend creating your own environment file (environment_sandbox) and source that.

Reply:
Oh... Good point.